Update Date Time property editor article (#6715)
umbraco/UmbracoDocs: 14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/date-time.md

![Data Type Definiton](images/date-time.png)

There is one setting available for manipulating the DateTime property.

> The setting involved defining the format. By default the format of the date in the Umbraco backoffice will be `YYYY-MM-DD HH:mm:ss`, but you can change this to something else. See [MomentJS.com](https://momentjs.com/) for the supported formats.

Reviewer: ```suggestion
The setting involves defining the format. The default date format in the Umbraco backoffice is `YYYY-MM-DD HH:mm:ss`, but you can change it to a different format. See [MomentJS.com](https://momentjs.com/) for the supported formats.
```
